addons/test_mail/tests/test_mail_message.py

```diff
@@ -373,3 +373,21 @@ def test_message_link_by_employee(self):
             self.assertEqual(res.url, expected_url)
         with self.subTest(deleted_message=deleted_message):
             res = self.url_open(f'/mail/message/{deleted_message.id}')
+
+@tagged("mail_message")
```

Comment:
Just to tag store tests and be sure we check for overrides
```suggestion
@tagged("mail_message", "mail_store", post_install", "-at_install)
```